### PR TITLE
COMP: Address brace initializer warning

### PR DESCRIPTION
**Pull request by blowekamp**
## PR Checklist
- [ ] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [ ] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [ ] Added test (or behavior not changed)
- [ ] Updated API documentation (or API not changed)
- [ ] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [ ] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [ ] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKExamples) for all new major features (if any)

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

**Review by hjmjohnson — APPROVED** [review]

**Comment by N-Dekker** [comment]
Looks fine Brad, but the original code also seemed fine to me! Which compiler did complain? And what was the message exactly?

**Comment by blowekamp** [comment]
https://open.cdash.org/viewBuildError.php?type=1&buildid=6930635

File: Modules/IO/ImageBase/test/itkWriteImageFunctionGTest.cxx [hunk 1]
```diff
@@ -36,13 +36,15 @@ struct ITKWriteImageFunctionTest : public ::testing::Test
...
 
   ImageType::Pointer
   MakeImage()
   {
     auto image = ImageType::New();
 
-    ImageType::RegionType region({ 3, 2 });
+    RegionType region(SizeType{ { 3, 2 } });
 
     image->SetRegions(region);
```
Comment (N-Dekker) [inline Modules/IO/ImageBase/test/itkWriteImageFunctionGTest.cxx:49]: You might also directly pass the size to image->SetRegions, as there is an overload, `SetRegions(const SizeType & size)`.

**Comment by N-Dekker** [comment]
> https://open.cdash.org/viewBuildError.php?type=1&buildid=6930635

Thanks Bradly.  I see now, from Mac10.10-AppleClang-dbg-x86_64-static:

```
[CTest: warning matched] /Users/builder/externalModules/IO/ImageBase/test/itkWriteImageFunctionGTest.cxx:45:36: warning: suggest braces around initialization of subobject [-Wmissing-braces]
    ImageType::RegionType region({ 3, 2 });
                                   ^~~~
                                   {   }

```

It suggests that it would be sufficient to just add an extra pair op braces. But your PR is also OK to me.

**Review by N-Dekker — APPROVED** [review]

**Review by thewtex — APPROVED** [review]

**Pull request merged**